server/config/incident.js

```diff
@@ -6,9 +6,21 @@ const removeEmptyValues = attrs => (inputs = []) => inputs.filter(hasAtLeastOneO
 
 const hasAtLeastOneOf = attrs => input => attrs.find(attr => !isBlank(input[attr]))
 
-const toBoolean = val => (val == null ? null : val === 'true')
+const toBoolean = val => {
+  switch (val) {
+    case 'true':
+      return true
+    case 'false':
+      return false
+    default:
+      return null
```

Comment:
Feels a bit wrong to be returning null here, I'm sure it makes sense given the right context.

Reply:
The form processing will remove the field if its null and not persist it in the DB. 
The changes here are to allow us to distinguish between true, false and ~FileNotFound~ not provided.